### PR TITLE
Admin Test Update Material

### PR DESCRIPTION
**Pull request by vhhieu95**
Admin Test Update Material

### CODE REVIEW

File: tests/Browser/Pages/Materials/UpdateMaterialTest.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+class UpdateMaterialTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+     
+    /**
+     * Test link UpdateMaterial.
+     *
+     * @return void
+     */
+    public function testUpdateFood()
```
Comment (ghost) [inline tests/Browser/Pages/Materials/UpdateMaterialTest.php]: Please make correct function name. I don't see update action in this function. Only check access to update page

File: tests/Browser/Pages/Materials/UpdateMaterialTest.php [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                    ->type('price', '')
+                    ->type('description', '')
+                    ->press('Save Changes')
+                    ->assertPathIs('/materials/2/edit')
+                    ->assertSee('The name field is required.')
+                    ->assertSee('The price field is required.')
+                    ->assertSee('The description field is required.');
+        });
+    }
+
```
Comment (ghost) [inline tests/Browser/Pages/Materials/UpdateMaterialTest.php:57]: Please add case update a material with a name already has in the database.

**Review by ghost — COMMENTED** [review]
Please check my comment.

**Comment by vhhieu95** [comment]
I updated, please check again.Thank you

**Comment by ghost** [comment]
LGTM 🐰

**Pull request merged**